packages/frontend/src/pages/scaling-tvl/view/ScalingTvlView.tsx

```diff
@@ -29,13 +36,25 @@ export interface ScalingTvlViewEntry {
   technology: string
 }
 
+export const UNVERIFIED_LIGHT_CX =
+  'bg-red-100 hover:bg-red-100 border-b-red-200'
+export const UNVERIFIED_DARK_CX =
+  'dark:bg-red-900 dark:hover:bg-red-900 dark:border-b-red-200'
```

Comment:
1. These two are never used separately, always together. For that reason there is no point in exporting them separately,
2. I don't like this being exported from here since it's not the responsibility of this component.

Export this as `getRowVerificationClassNames(entry: { isVerified?: boolean }): string` from `components/table/getRowVerificationClassNames.ts` and encapsulate the whole logic of checking `isVerified` there.